### PR TITLE
Fix/autocomplete api debug

### PR DESCRIPTION
**Pull request by pcmueller**
## Type of Change
- [x] JSX
- [ ] HTML
- [ ] JavaScript
- [ ] CSS
- [x] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [x] Refactor/Fix Bugs

## Please describe the change made
- Connected new .env file / API key to SearchLocationInput, now properly autocompleting location inputs
- Switched "cities" location type to "address" - now properly searching for addresses
- Added second location input to SearchLocationInput, then disabled it.  We can talk more in person, but essentially both inputs feed to the same address variable in state ("query"), and since that variable also connects to the Google API request, we can't just add a second one.  Can't do a separate component altogether - it just gives us the same old "multiple API per page" error.
- Disabled page-load display of ETA modal.

## How has this been tested?
- [x] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

File: src/Components/Form/Form.js [hunk 1]
```diff
@@ -36,17 +38,9 @@ function Form({contacts}) {
...
-        name='input'
-        className='start-point'
-        placeholder='Start Point'
-      />
-      <input
-        type='text'
-        name='input'
-        className='end-point'
-        placeholder='End Point'
+      <SearchLocationInput
```
Comment (bhohnco) [inline src/Components/Form/Form.js]: Nice work here. We'll have to research how to work around the API issue and what our options are.

File: src/Components/SearchLocationInput/SearchLocationInput.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <input
+        ref={autoCompleteRef}
+        onChange={event => setQuery(event.target.value)}
+        placeholder="Starting address"
+        value={query}
+        className='location-input'
+        required
+      />
+      <input
+        // ref={autoCompleteRef}
```
Comment (bhohnco) [inline src/Components/SearchLocationInput/SearchLocationInput.js]: Going to look into this tonight and see what I find. I'll circle back once I know more.

Comment (pcmueller) [reply src/Components/SearchLocationInput/SearchLocationInput.js]: Sounds good - I'll do the same tomorrow.

File: src/Components/Form/Form.js [hunk 3]
```diff
@@ -40,21 +45,30 @@ function Form({contacts}) {
...
-        type='text'
-        name='input'
-        className='end-point'
-        placeholder='End Point'
+      <Autocomplete
+          onPlaceSelected={(place,) => {
+            console.log(place);
+          }}
+          onChange={event => setQuery(event.target.value)}
+          options={{types: ["address"]}}
```
Comment (carolineshowscode) [inline src/Components/Form/Form.js:63]: Good solution utilizing the types property to autocomplete the full address

**Review by carolineshowscode — APPROVED** [review]
Nice work, CI is passing. Next we can work on passing the Autocomplete address prop with the onPlaceSelected listener created on 49-50 & 59-60

**Pull request merged**